src/crunch/crn_decomp.rs

```diff
@@ -0,0 +1,347 @@
+#![allow(non_snake_case)]
```

Comment:
It might be better to simply convert all non xyz case types to the normal Rust convention.
-> simple refractoring via any IDE